crypto/tests/test_check.c

```diff
@@ -8307,6 +8307,30 @@ START_TEST(test_rc4_rfc6229) {
 }
 END_TEST
 
+START_TEST(test_compress_coords) {
+  const ecdsa_curve *curve = &secp256k1;
+  curve_point expected_coords;
+
+  random_reseed(time(NULL));
+  bignum256 k = {0};
+  while (bn_is_zero(&k)) {
```

Comment:
Please provide a set (10?) of static vectors, not randomly generated ones.

Reply:
> Please provide a set (10?) of static vectors, not randomly generated ones.

done